pkg/rest/tests/rest_test.go

```diff
@@ -20,22 +21,28 @@ import (
 )
 
 var (
-	httpClient      = &feederfakes.FakeHttpClient{}
-	failHttpClient  = &feederfakes.FailHttpClient{}
-	client          *feeder.Client
-	failClient      *feeder.Client
-	restHandler     = rest.RestHandler{}
-	failRestHandler = rest.RestHandler{}
+	httpClient         = &feederfakes.FakeHttpClient{}
+	failHttpClient     = &feederfakes.FailHttpClient{}
+	client             *feeder.Client
+	failClient         *feeder.Client
+	restHandler        = rest.RestHandler{}
+	failRestHandler    = rest.RestHandler{}
+	failRequestTimeout = time.Millisecond * 400
+	baseUrl            = "https://localhost:8100"
+	baseApi            = "/feeder_gateway/"
```

Comment:
Use consistent case for acronyms i.e. `baseURL` instead of `baseUrl`. See [guide](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).